embedding/csharp/Tizen.Flutter.Embedding/FlutterApplication.cs

```diff
@@ -93,28 +87,10 @@ protected override void OnCreate()
         {
             base.OnCreate();
 
-            Utils.ParseEngineArgs(EngineArgs);
-
-            using (var switches = new StringArray(EngineArgs))
-            using (var entrypointArgs = new StringArray(DartEntrypointArgs))
+            Engine = new FlutterEngine(DartEntrypoint, DartEntrypointArgs);
+            if (!Engine.IsValid)
             {
-                var engineProperties = new FlutterDesktopEngineProperties
-                {
-                    assets_path = "../res/flutter_assets",
-                    icu_data_path = "../res/icudtl.dat",
-                    aot_library_path = "../lib/libapp.so",
-                    switches = switches.Handle,
-                    switches_count = (uint)switches.Length,
-                    entrypoint = DartEntrypoint,
-                    dart_entrypoint_argc = entrypointArgs.Length,
-                    dart_entrypoint_argv = entrypointArgs.Handle,
-                };
-
-                Engine = FlutterDesktopEngineCreate(ref engineProperties);
-                if (Engine.IsInvalid)
-                {
-                    throw new Exception("Could not create a Flutter engine.");
-                }
+                throw new Exception("Could not create a Flutter engine.");
```

Comment:
Inside FlutterEngine() , it already checks `Engine.IsInvalid` and throws an exception with the same message.
I think we can reduce the exception throw or send a different message.

Reply:
Good point! I changed the constructor not to throw an exception. How do you like? Now it's the user's responsibility to check if the created `FlutterEngine` is valid or not.